packages/server/src/main/java/software/uncharted/terarium/hmiserver/service/notification/NotificationGroupInstance.java

```diff
@@ -107,10 +117,22 @@ public void sendMessage(final String msg) {
 		sendNotification(msg, null, estimateT(), ProgressState.RUNNING);
 	}
 
+	public void sendMessage(final String msg, ProgressState state) {
+		sendNotification(msg, null, estimateT(), state);
+	}
+
 	public void sendFinalMessage(final String msg) {
 		sendNotification(msg, null, 1.0, ProgressState.COMPLETE);
 	}
 
+	public void sendFinalMessage(final String msg, ProgressState state) {
```

Comment:
Stylistically seems like `ProgressState` should be final as well? ditto for the sendMessage method.

Reply:
Good catch! Fixed it.